reactors/1D_packbed.ipynb

```diff
@@ -0,0 +1,518 @@
+    "\n",
+    "If a perm-selective membrane is present, then $j_{k_{\\rm M}}$ represents the gas-phase flux through the membrane and ${k_{\\rm M}}$ is the gas-phase species that permeates through the membrane. The Kronecker delta, $\\delta_{k, k_{\\rm M}} = 1$ for the membrane-permeable species and  $\\delta_{k, k_{\\rm M}} = 0$ otherwise. The membrane flux is calculated using Sievert's law as\n",
+    "$$\n",
+    "    j_{k_{\\rm M}}^{\\rm{Mem}} = \\frac{B_{k_{\\rm M}}}{t} \\left ( p_{k_{\\rm M}, {\\rm{mem}}}^\\alpha - p_{k_{\\rm {M,                 sweep}}}^\\alpha \\right ) W_{k_{\\rm M}}\n",
+    "$$\n",
+    "where $B_{k_{\\rm M}}$ is the membrane permeability, $t$ is the membrane thickness. $p_{k_{\\rm M}, {mem}}$ and $p_{k_{\\rm {M, sweep}}}$ represent perm-selective species partial pressures within the packed-bed and the exterior sweep channel. The present model takes the pressure exponent $\\alpha$ to be unity. The membrane flux for all other species ($ k \\neq k_{\\rm M}$) is zero.\n",
+    "\n",
+    "### Chemistry mechanism\n",
+    "This example uses a detailed 12-step elementary micro-kinetic reaction mechanism that describes ammonia formation and decomposition kinetics over the Ru/Ba-YSZ catalyst. The reaction mechanism is developed and validated using measured performance in a laboratory-scale packed-bed reactor[3]. This example also incorporates a Pd-based H$_2$ perm-selective membrane.\n",
+    "\n",
+    "\n",
+    "### Solver\n",
+    "\n",
+    "Above governing equations represent a complete solution for a steady-state packed-bed, catalytic membrane reactor model. The dependent variables are the mass-flux $\\rho u$, species mass-fractions $Y_k$, pressure $p$, temperature $T$, and surface coverages $Z_k$. The equation of state is used to obtain the mass density, $\\rho$.\n",
+    "\n",
+    "The governing equations form an initial value problem (IVP) in a differential-algebraic (DAE) form as follows-\n",
+    "$$\n",
+    "   f(z,{\\bf{y}}, \\bf {y'}, c) = 0,\n",
+    "$$\n",
+    "where $\\bf{y}$ and $\\bf{y'}$ represent the solution vector and its derivative vector, respectively. All other constants such as reference temperature, chemical constants, etc. are incorporated in vector $c$ (Refer [4] for more details). This type of DAE system in this example is solved using scikits.odes.dae IDA solver.\n",
```

Comment:
```suggestion
    "where $\\bf{y}$ and $\\bf{y'}$ represent the solution vector and its derivative vector, respectively. All other constants such as reference temperature, chemical constants, etc. are incorporated in vector $c$ (refer to [4] for more details). This type of DAE system in this example is solved using the `scikits.odes.dae` IDA solver.\n",
```
What does IDA stand for?

Reply:
Corrected. IDA is a package from Sundials for solving a system of differential-algebraic equation (DAE).